AgentAssistAngular/src/home/components/home/home.component.ts

```diff
@@ -99,6 +99,29 @@ export class HomeComponent implements OnInit {
     }, 5000);
   }
 
+  hintAckPressed(data) {
+    console.log('Hints Data here: ', data)
+    let agent_coaching_ackpress = {
+    "botId": this.connectionDetails.botId,
+    "event": data?.event,
```

Comment:
@SandeepBa-Kore why are you adding this whole payload, you just need to add **ackPressed** true in **data** object